extensions/wikia/Forum/css/venus/RelatedForumDiscussion.scss

```diff
@@ -1,5 +1,4 @@
 @import "skins/shared/color";
-@import "extensions/wikia/Venus/styles/variables";
 
 .RelatedForumDiscussion {
```

Comment:
Do we need this file at all? If yes, shouldn't it be moved away from `venus` directory?